packages/playground/cli/src/run-cli.ts

```diff
@@ -263,7 +263,7 @@ export async function parseOptionsAndRunCLI() {
 						new URL(args.wp);
 					} catch {
 						throw new Error(
-							'Unrecognized WordPress version. Please use "latest", a URL, or a numeric version such as "6.2", "6.0.1", "6.2-beta1", or "6.2-RC1"'
+							'Unrecognized WordPress version. Please use "latest", "beta", "trunk", "nightly", a URL, or a numeric version such as "6.2", "6.0.1", "6.2-beta1", or "6.2-RC1"'
```

Comment:
@adamziel I think the point is here, the conditional check if the version is valid, if not skip it and doesn't do anything. I think we should throw a message if the value is not valid

```
if (args.wp !== undefined && !isValidWordPressSlug(args.wp)) {
					try {
						// Check if is valid URL
						new URL(args.wp);
					} catch {
						throw new Error(
							'Unrecognized WordPress version. Please use "latest", "beta", "trunk", "nightly", a URL, or a numeric version such as "6.2", "6.0.1", "6.2-beta1", or "6.2-RC1"'
						);
					}
				}
				```